python/dronin-logfsimport

```diff
@@ -12,10 +12,10 @@ def main():
 
```

Comment:
I thought about it but I thought the line would get too long... plus this bit of functionality isn't intended for end users-- it's just there for the cloud (and the split was added here so I could try it out).